src/dmtcpplugin.cpp

```diff
@@ -55,7 +55,7 @@ memfence() {  asm volatile ("mfence" ::: "memory"); }
 static inline void
```

Comment:
Please merge this commit into the previous commit:
> iscv: Defining inline functions RMB, WMB and IMB

(I have a review comment there saying that these two codes are related.)